recipe/bld.bat

```diff
@@ -1,6 +1,8 @@
 if not exist %PREFIX% mkdir %PREFIX%
+if errorlevel 1 exit /b 1
```

Comment:
Would there be an `errorlevel` if we haven't run anything? Think that may be why CI is failing (as it doesn't seem to be pointing to this line

If we do want this, we may want as part of the `if` right after `mkdir`